samples/Directory.Build.props

```diff
@@ -0,0 +1,9 @@
+<Project>
+
+  <Import Project="../Directory.Build.props" />

```

Comment:
MSBuild scans directories upwards from the project file, and stop as soon as it encounters a DIrectory.Build.props file. To also import the Directory.Build.props file from the parent directory, we have to do it explicitly. Note that the approach mentioned [here](https://docs.microsoft.com/en-us/visualstudio/msbuild/customize-your-build#use-case-multi-level-merging) doesn't seem to work; this one is simpler anyway...